lib/database/adapters/LhcPeriodAdapter.js

```diff
@@ -29,8 +29,15 @@ class LhcPeriodAdapter {
      * @param {SequelizeLhcPeriod} databaseObject Object to convert.
      * @returns {LhcPeriod} Converted entity object.
      */
-    toEntity({ id, name }) {
-        return { id, name };
+    toEntity(databaseObject) {
+        const { id, name } = databaseObject;
+        const { runs, dataPasses } = databaseObject;
+        const runsCount = runs ? runs[0]?.get('count') ?? 0 : null;
```

Comment:
Again, it is not clear to me why you have a count in an item. I think it makes the code very hard to understand, and I'm pretty sure there is a way to do otherwise.
If not, please at least put a comment on why you do it this way, and an info about it in the docblock.

Reply:
does not apply anymore